### PR TITLE
feat: reproducibility graph

### PR DESCRIPTION
**Pull request by tdejager**
Bring back the html graph.

image

### CODE REVIEW

File: src/repror/internals/db.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import os
 from pathlib import Path
 import tempfile
-from typing import Generator, Optional
+from typing import Generator, Literal as Lit, Optional
```
Comment (nichmor) [inline src/repror/internals/db.py:10]: why Literal as Lit?

File: src/repror/cli/generate_html.py [hunk 2]
```diff
@@ -94,9 +95,28 @@ def rerender_html(root_folder: Path, update_remote: bool = False):
...
+    start = datetime.now() - timedelta(days=9)
+    end_of_day = datetime(start.year, start.month, start.day, 23, 59, 59)
+    timestamps = [end_of_day + timedelta(days=i) for i in range(0, 10)]
     by_platform = defaultdict(list)
-
     for build in builds:
+
+        # Do it in the loop so that we do this only once per platform
+        # and we dont have to hardcode the platforms
+        if not counts_per_platform.get(build.platform_name):
```
Comment (nichmor) [inline src/repror/cli/generate_html.py:109]: I think it's more pythonic to write this way:
`if build.platform_name not in counts_per_platform` as we can't have an empty value for key

**Review by nichmor — APPROVED** [review]
I like it! 📈

File: src/repror/internals/db.py (hunk 1 above)
Comment (tdejager) [reply src/repror/internals/db.py:10]: Because there is another Literal import beneath it.

**Pull request merged**